scripts/lib/installer-win32-gen.tsx

```diff
@@ -151,7 +155,91 @@ function getDescendantDirs(d: directory): directory[] {
  */
 export default async function generateFileList(rootPath: string): Promise<string> {
   const rootDir = await walk(rootPath);
-  const descendantDirs = getDescendantDirs(rootDir);
+  const descendantDirs = getDescendantDirs(rootDir).filter(d => d.files.length > 0);
+
+  const specialComponents: Record<string, (d: directory, f: { name: string, id: string }) => Element> = {
+    'Rancher Desktop.exe': (d, f) => {
```

Comment:
I noticed `d` is not used at all, is it just here to satisfy the expression signature for the next entry (line:191)?

Reply:
Yep.